README.md

```diff
@@ -77,6 +96,9 @@ spec = APISpec(
 spec.components.schema('Category', schema=CategorySchema)
 spec.components.schema('Pet', schema=PetSchema)
 # pass created resource into `path` for APISpec
+# should be passed twice so the suffix is registered
+# path should be called n + 1 times where n is the number of suffixes
```

Comment:
Unless people read this README, they are not going to know about this convention.

Is there a better way to register resources? Would prefer to do it once versus `n+1` times. Also, is there is a reason it's `n+1`?

Reply:
Hi, Unfortunately not, due to how the ApiSpec plugin works. It calls each resource once, meaning if you use a controller twice only one path will be chosen.

Reply:
@alysivji Any updates?

Reply:
Calling something n+1 times is a sign of bad design. What if y'all add a new suffix but forgot to add the line? Having a manual process is a recipe for disaster.

There has to be a way of fixing it I would need to look into how the path_helper works... why can't we loop n+1 times inside there to get the information you need?

I have commitments until August 4th. I can definitely help out after then.

Reply:
@alysivji Status?